### DISCUSSION TITLE
Wrapped in (T)ISSUE Paper 🎁 [Issues Mini Check-in]

### DISCUSSION THREAD

**Discussion by queenofcorgis** [community/community#40579] [Projects and Issues]
The gift of a new month also brings the gift of fresh mini check-ins full of shoutouts, releases, and more! Unwrap the latest with us 💝

🦸‍♂️ **Incredible in Issues:** A round of applause for these community citizens 👏
* @conorriches: For an awesome enhancement! They [suggested](https://github.com/orgs/community/discussions/38557) that the title of an Issue remain visible even when scrolling in overlay view. This has now shipped 🚢! 
* @Makuna: Thank you submitting your feature request around Issues and Discussions being confusing to customers. Does anyone have any suggestions for how they do this in their repos? [Chime in now!](https://github.com/orgs/community/discussions/40363) 

🐙**Issues Info**: All updates, info, releases, etc that are Issues-related!
* **Latest Update**: Fresh out of Universe, you can convert drafts, automatically link from index pages, and bug fixes. [Learn more](https://github.blog/changelog/2022-11-17-github-issues-november-17th-update/).  

📗**Rousing Reads**: GitHub blog selected posts just for you! 
* **The journey of your work has never been clearer**:  @mariorod announces even more features designed to make it easier to plan and track in the same place you build in [his post](https://github.blog/2022-11-15-the-journey-of-your-work-has-never-been-clearer/)!

Are you as excited as I am for all these new features?! Which ones have you most excited about - I want to know!

**Comment by JVimes** [comment]
I look at the Projects feature once in a while. I still can't drag multi-selected cards around on Boards. I think simple features got passed up for advanced ones. I'm not sure Boards view is well supported. I hope it becomes more viable soon. I wish you well.

**Reply by queenofcorgis** [reply]
Thanks for the feedback @JVimes - have you created a Discussion post with that sentiment? That's the best way to ensure it is going to the teams working on Projects! 

Appreciate you chiming in!

**Reply by JVimes** [reply]
Thank you @queenofcorgis. I upvoted on existing discussions ([1][1], [2][2]) but I'm not sure they are influencing the product. Should I start a new discussion?

[1]: https://github.com/orgs/community/discussions/6528#discussioncomment-1504100
[2]: https://github.com/orgs/community/discussions/8960

**Reply by conorriches** [reply]
You could drop a comment on the existing issues so they're bumped to the top of the list for others to view.

Bumping them with a comment that adds something to the discussion can absolutely help get them seen and further refined.

**Reply by queenofcorgis** [reply]
Well said @conorriches - feel free to add any comments on the functionality from your perspective, helps our team understand use cases and potential @JVimes!

**Reply by JVimes** [reply]
Thanks! I didn't think about it that way. Appreciate it.
